packages/react-scripts/template/src/index.css

```diff
@@ -1,3 +1,8 @@
+/**
+ * This is a good place to put styles that apply to the whole page.
+ * We recommend to put component-specific styles into separate files.
```

Comment:
Super nit picky but "We recommend to put" sounds a bit odd, how about "We recommend putting"